ibis/tests/sql/snapshots/test_select_sql/test_case_in_projection/decompiled.py

```diff
@@ -0,0 +1,40 @@
+import ibis
+
+
+l = ibis.literal("foo")
```

Comment:
Not a huge fan of variable names that start with `l`, but it's generated code so 🤷🏻

Reply:
Renamed it to `lit` for now